### PR TITLE
feat: using device unique identifier on unleash and screen to share it

### PR DESCRIPTION
**Pull request by andreabadesso**
# Acceptance criteria

The user must be able to share his device's unique identifier and Hathor team should be able to use it to fix his device on the new wallet facade

### CODE REVIEW

File: src/actions.js [hunk 1]
```diff
@@ -237,8 +243,8 @@ export const startWallet = (words, pin) => async (dispatch) => {
   walletUtil.cleanLoadedData();
 
   const networkName = 'mainnet';
-  const firstAddress = getFirstAddressFromWords(words, networkName);
```
Comment (pedroferreira1) [inline src/actions.js:240]: Can we remove `getFirstAddressFromWords` method?

**Review by pedroferreira1 — APPROVED** [review]

File: src/actions.js (hunk 1 above)
Comment (andreabadesso) [reply src/actions.js:240]: Yes we can, done in 0ed36ff319aa50ca4e96c0c564c5e7d3f27eca30

File: src/actions.js [hunk 2]
```diff
@@ -353,7 +359,7 @@ export const fetchNewTxTokenBalance = async (wallet, tx) => {
   const updatedBalanceMap = {};
   const balances = wallet.getTxBalance(tx);
   // we now loop through all tokens present in the new tx to get the new balance
-  for (const [tokenUid, tokenTxBalance] of Object.entries(balances)) {
+  for (const [tokenUid, _tokenTxBalance] of Object.entries(balances)) {
```
Comment (r4mmer) [inline src/actions.js:362]: if `_tokenTxBalance` is never used, wouldn't a `_` be better?

**Review by r4mmer — APPROVED** [review]

File: src/actions.js (hunk 2 above)
Comment (andreabadesso) [reply src/actions.js:362]: I think the effect is the same (making it clear that the variable is unused)

I prefer leaving the variable name there so we know what it is

**Pull request merged**